### PR TITLE
Update Metrics document - Config.md

### PR DESCRIPTION
**Pull request by rajkumar-rangaraj**
Fixes doc for issue #662 

Update documents for Metrics in Config.md

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -25,12 +26,14 @@ for more details.
 | Environment variable | Description | Default value |
 |-|-|-|
 | `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE` | File path of JSON configuration files of bytecode instrumentations. For example: `%ProfilerDirectory%/integrations.json` | |
-| `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` | Comma-separated list of source instrumentations you want to enable. |  |
-| `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS` | Comma-separated list of source and bytecode instrumentations you want to disable. | |
+| `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` | Comma-separated list of traces source instrumentations you want to enable. |  |
+| `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS` | Comma-separated list of traces source and bytecode instrumentations you want to disable. | |
+| `OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS` | Comma-separated list of metrics source instrumentations you want to enable. |  |
+| `OTEL_DOTNET_AUTO_METRICS_DISABLED_INSTRUMENTATIONS` | Comma-separated list of metrics source and bytecode instrumentations you want to disable. | |
```
Comment (pellared) [inline docs/config.md:32]: I do not think it is not entitely true. This setting works only for source instrumentation.

File: docs/config.md [hunk 2]
```diff
@@ -134,10 +145,13 @@ Important environment variables include:
 | Environment variable | Description | Default value |
 |-|-|-|
 | `OTEL_DOTNET_AUTO_LOAD_TRACER_AT_STARTUP` | Whether the tracer is created by the automatic instrumentation library or not. Set to `false` when the application initializes the OpenTelemetry .NET SDK Tracer on its own. This configuration can be used, for example, to retrieve the bytecode instrumentations. | `true` |
+| `OTEL_DOTNET_AUTO_LOAD_METER_AT_STARTUP` | Whether the meter is created by the automatic instrumentation library or not. Set to `false` when the application initializes the OpenTelemetry .NET SDK Meter on its own. This configuration can be used, for example, to retrieve the bytecode instrumentations. | `true` |
```
Comment (nrcventura) [inline docs/config.md:140]: We mention `This configuration can be used, for example, to retrieve the bytecode instrumentations.` in the equivalent setting for tracing, but I'm confused by it. Maybe we should remove it?

Comment (rajkumar-rangaraj) [reply docs/config.md:140]: I agree, we should remove it. This is kept for future use.

Comment (rajkumar-rangaraj) [reply docs/config.md]: Fixed 9f1a571d2878ee3bc343b579ff3b343ffd30b87d

**Review by nrcventura — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

File: docs/config.md [hunk 3]
```diff
@@ -25,12 +26,14 @@ for more details.
 | Environment variable | Description | Default value |
 |-|-|-|
 | `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE` | File path of JSON configuration files of bytecode instrumentations. For example: `%ProfilerDirectory%/integrations.json` | |
-| `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` | Comma-separated list of source instrumentations you want to enable. |  |
-| `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS` | Comma-separated list of source and bytecode instrumentations you want to disable. | |
+| `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` | Comma-separated list of traces source instrumentations you want to enable. |  |
+| `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS` | Comma-separated list of traces source and bytecode instrumentations you want to disable. | |
+| `OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS` | Comma-separated list of metrics source instrumentations you want to enable. |  |
+| `OTEL_DOTNET_AUTO_METRICS_DISABLED_INSTRUMENTATIONS` | Comma-separated list of metrics source instrumentations you want to disable. | |
```
Comment (pellared) [inline docs/config.md:32]: What is our plan regarding bytecode instrumentation for metrics?

**Review by pellared — APPROVED** [review]

File: docs/config.md (hunk 3 above)
Comment (pellared) [reply docs/config.md:32]: I just want to discuss how we want to do it in the future. For example, how we would add bytecode metrics instrumentation to GraphQL? It is not required at all for the next release.

**Pull request merged**